### ISSUE TITLE
Material Keyboard Navigation Duplicated

### ISSUE THREAD

**Issue by emilygb-star** [openlibhums/janeway#4917]
2.4.7 Focus Visible

Head point: c96d43d19

Navigation point: all pages

When navigating by keyboard on Chrome desktop view, with the theme ``material``, you must navigate through the nav bar (Home, Articles, Issues, etc.) twice before reaching the main page components below. When navigating through the components a second time, no focus is indicated; perhaps the narrow screen navigation is being included. This is true for navigating all pages in this theme.
